client/src/app/pages/dependencies/dependency-apps-table.tsx

```diff
@@ -285,11 +285,21 @@ const DependencyVersionColumn = ({
 }: {
   appDependency: AnalysisAppDependency;
 }) => {
-  const isJavaDependency = name && version && sha && provider === "java";
+  let mavenCentralLink;
 
-  const mavenCentralLink = isJavaDependency
-    ? `https://search.maven.org/search?q=1:${extractFirstSha(sha)}`
-    : undefined;
+  if (name && version && provider === "java") {
+    if (sha) {
+      mavenCentralLink = `https://search.maven.org/#search|1:${extractFirstSha(
+        sha
+      )}`;
+    } else {
+      const group = name.substring(0, name.lastIndexOf("."));
```

Comment:
are we sure that the name is in correct format? if not then I would go with sth like:
```
const [group, artifact] = name.split(".");
if ( group && artifact) {
    // continue
} else {
  // ??
}
```

Reply:
For a java dependency from maven, the dependency name will be group + artifactid.  The end point looks like:
```sh
curl http://192.168.39.190/hub/analyses/report/dependencies/applications
```

With an output that looks like:
```json
[
  {
    "id": 5,
    "name": "Bear",
    "description": "Bear application, oh my!.",
    "businessService": "",
    "dependency": {
      "id": 18,
      "provider": "java",
      "name": "javax.xml.bind.jaxb-api",
      "version": "2.3.1",
      "sha": "c42c51ae84892b73ef7de5351188908e673f5c69",
      "indirect": true,
      "labels": [
        "konveyor.io/dep-source=open-source",
        "konveyor.io/language=java"
      ]
    }
  },
  {
    "id": 5,
    "name": "Bear",
    "description": "Bear application, oh my!.",
    "businessService": "",
    "dependency": {
      "id": 19,
      "provider": "java",
      "name": "net.bytebuddy.byte-buddy",
      "version": "1.10.22",
      "sha": "14de25cfee49cd27ae19153674bbb34c04c45d52",
      "indirect": true,
      "labels": [
        "konveyor.io/dep-source=open-source",
        "konveyor.io/language=java"
      ]
    }
  },
  {
    "id": 5,
    "name": "Bear",
    "description": "Bear application, oh my!.",
    "businessService": "",
    "dependency": {
      "id": 20,
      "provider": "java",
      "name": "org.apache.logging.log4j.log4j-api",
      "version": "2.14.1",
      "sha": "9199a73770616b1ca0b00f576db3231aaab4876a",
      "indirect": true,
      "labels": [
        "konveyor.io/dep-source=open-source",
        "konveyor.io/language=java"
      ]
    }
  },
]
```

The `group.artifactId` parsing is ok.